build/.gitignore

```diff
@@ -1,4 +0,0 @@
-# Ignore everything in this directory
```

Comment:
Please recover this file.

Reply:
Correct. I guess you used to remove build folder completely when you start a new build. However, the `build` folder of QMCPACK is more for display and we have a hidden file.